yt_dlp/webvtt.py

```diff
@@ -95,6 +95,7 @@ def __init__(self, parser):
 _REGEX_EOF = re.compile(r'\Z')
 _REGEX_NL = re.compile(r'(?:\r\n|[\r\n]|$)')
 _REGEX_BLANK = re.compile(r'(?:\r\n|[\r\n])+')
+_REGEX_SPACE = re.compile(r'(?: )+')
```

Comment:
```suggestion
_REGEX_OPTIONAL_WHITESPACE =re.compile(r'[ \t]*')
```
The spec optionally allows tabs and spaces in front of the settings list.
See: https://www.w3.org/TR/webvtt1/#webvtt-cue-block